main.cpp

```diff
@@ -722,15 +722,15 @@ int main(int argc, char *argv[])
 {
     Mainloop &mainloop = Mainloop::init();
 
-    log_open();
+    Log::open();
```

Comment:
We will need the open/close whenever we are not using only stderr for logging. For example when/if we integrate it with the journal.